configure

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
```

Comment:
```suggestion
#!/usr/bin/env sh
```

Reply:
Just wondering: why change this?

Reply:
Merge is being blocked because of this. @XPhyro can you approve (assuming you're fine with the PR)?